cpp/data/system_updates/102.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -o errexit
+
+
+if [[ $TUEFIND_FLAVOUR != "krimdok" ]]; then
```

Comment:
If it is not krimdok, then it can either be ixtheo (like ptah/ub16) or ub_tools_only (like nu/ub28). Do you really want it to run on both machine types?